opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/FileManager.java

```diff
@@ -626,7 +626,7 @@ public void matchUpVariantFiles(String studyStr, List<File> transformedFiles, St
             }
             File file = new File("", createParams.getType(), createParams.getFormat(), createParams.getBioformat(), null, path, "",
                     createParams.getCreationDate(), createParams.getModificationDate(), createParams.getDescription(), false, 0,
-                    createParams.getSoftware(), null, createParams.getSampleIds(), null, "", 1, createParams.getTags(), null, null, null,
+                    createParams.getSoftware(), null, createParams.getSampleIds(), null, createParams.getJobId(), 1, createParams.getTags(), null, null, null,
```

Comment:
@pfurio , to fix one ExecutionDaemonTest, I had to add this here, which kind of "makes sense". Do you agree?

Reply:
Yes, it does